2020q3/linuxulator.md

```diff
@@ -0,0 +1,38 @@
+## Linux compatibility layer update ##
+
+Contact: Edward Tomasz Napierala, <trasz@FreeBSD.org>
+
+Earlier Linuxulator work focused on code cleanups and improving
+diagnostic tools.
+Work has now shifted from cleanups to fixing actual applications.
+Current status is being tracked at
+[Linux app status Wiki page](https://wiki.freebsd.org/LinuxApps).
+Initial focus was on applications that don't involve X11, mostly
+because they tend to be easier to test and debug, and the bug fixes
+are not application-specific.
+
+Foundation-sponsored work during this quarter included implementing
+a devfs(5) workaround to fix gettynam(3) inside jail/chroot, and
+workaround for the missing splice(2) syscall, which caused problems
+for grep and autotools. The Linux version reported to userspace was bumped
+to 3.10.0, which matches the kernel shipped with RHEL 7 and is neccessary
+for DB2 installation to succeed.  The BLKPBSZGET ioctl neccessary for
+Oracle is supported now.  There were also more cleanups, including moving
+some Linuxulator-specific functionality related to error handling off
+from the syscall's fast code paths.  The sysutils/debootstrap was updated
```

Comment:
and clarify what debboostrap is

Reply:
It also seems to be missing the word "port"